src/document.cpp

```diff
@@ -89,16 +89,13 @@ void REHex::Document::save()
 
 void REHex::Document::save(const std::string &filename)
```

Comment:
Changes to this method will leave filename and title incorrect if the write fails - they should only be updated if it completes, hence the duplication.

Reply:
reverted, I did indeed oversee the reasoning behind it.